.github/workflows/archive-extension-source-code.yml

```diff
@@ -0,0 +1,77 @@
+name: Archive Extension Build And Source Code
+
+on:
+  push:
+    tags:
+      - coil-extension@*
+
+jobs:
+  save-monorepo-source-code:
+    runs-on: ubuntu-latest
+   
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: Set env
+        run: echo "RELEASE_VERSION=${GITHUB_REF#refs/*/}" >> $GITHUB_ENV
+
+      - name: Step 1 - Tar files
```

Comment:
Add a comment with a link to the permissions issues

Reply:
Zip format would actually work fine (if you used a zip util), and would be preferable (you could download the zip inside a zip, extract the zip and upload it. with tar, the reviewer will have to extract the tar as an extra step) 

It's the upload/download actions that are the problem here.